packages/shared/src/utils/constants/wallets.ts

```diff
@@ -0,0 +1,202 @@
+// TODO: Refactor using data transformation layer pattern when we will have API for wallets (e.g. wallet.icon)
```

Comment:
@sergei-deriv `useWalletAccounts` 👀
You can use `useFetch` + `select` option, Let me know if you need help with that 🙏🏻